server/task/task-memory/src/main/java/org/apache/james/task/SerialTaskManagerWorker.java

```diff
@@ -94,14 +100,23 @@ public SerialTaskManagerWorker(Listener listener) {
                                 listener.failed(taskWithId.getId(), taskWithId.getTask().details(), exception);
                             }
                         })
-                        .onErrorReturn(Task.Result.PARTIAL);
+                        .onErrorReturn(Task.Result.PARTIAL)
+                        .doOnTerminate(informationPolling::dispose);
             } else {
                 listener.cancelled(taskWithId.getId(), taskWithId.getTask().details());
                 return Mono.empty();
             }
         };
     }
 
+    private Flux<TaskExecutionDetails.AdditionalInformation> pollAdditionalInformation(TaskWithId taskWithId) {
+        return Mono.fromCallable(() -> taskWithId.getTask().details())
+            .delayElement(Duration.ofSeconds(1))
```

Comment:
didn't we speak of 30s or 1mn?

Reply:
As it is in the code currently, we have 30s. 
And we use 1 s during the test